File: nelio-alves-algoritmo-e-logica-de-programacao/cpp/exercicios/media_idade.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    int idade, soma, cont;
    double media;
    soma = 0;
    cont = 0;

    cout << "Digite as idades:" << endl;
    cin >> idade;

    if (idade < 0) {
        cout << "IMPOSSIVEL CALCULAR" << endl;
    } else {
        while (idade >= 0) {
            soma = soma + idade;
            cont++;

            cin >> idade;
        }

        media = (double) soma / cont;

        cout << fixed << setprecision(2);
        cout << "MEDIA = " << media << endl;
    }

	return 0;
}

File: nelio-alves-algoritmo-e-logica-de-programacao/python/exercicios/media_idade.py
idade: int; soma: int; cont: float
media: float
soma = 0
cont = 0

print("Digite as idades:")
idade = int(input())

if idade < 0:
	print("IMPOSSIVEL CALCULAR")
else:
	while idade >= 0:
		soma = soma + idade
		cont = cont + 1

		idade = int(input())

	media = soma / cont

	print(f"MEDIA = {media:.2f}")